apps/docs/features/docs/Troubleshooting.ui.client.tsx

```diff
@@ -200,6 +202,25 @@ function TroubleshootingFilterInternal({
     allEntries.current = entries
   }, [])
 
+  const allErrorCodes = useMemo(
+    () =>
+      Array.from(
+        new Set(
+          errors?.flatMap((error) => {
+            const result = []
+            if (error.http_status_code) {
+              result.push(error.http_status_code)
+            }
+            if (error.code) {
+              result.push(error.code)
+            }
+            return result
+          }) ?? []
+        )
+      ),
+    [errors]
+  )
```

Comment:
Quick q - why create a `Set` then convert back to an array? Couldn't you return the result of the `flatMap` directly?

Reply:
To remove duplicates: we could have multiple error entries that return the same HTTP status code for example

Reply:
Gotcha!